assets/helpers/abtest.js

```diff
@@ -143,8 +155,8 @@ function getParticipation(mvtId: number): Participations {
 
 export const init = () => {
 
-  const mvt = getMvtId();
-  let participation = getParticipation(mvt);
+  const mvt: number = getMvtId();
```

Comment:
I made it explicit in order to be more readable for someone who is reading the code and doesn't know what Mvt is.